azure-pipelines.yml

```diff
@@ -125,6 +125,11 @@ stages:
             solution: "**/*.sln"
             feedstoUse: config
 
+        # Authenticate for npmrc
+        - task: npmAuthenticate@0
```

Comment:
We need to authenticate to public npm feeds? that's kind of annoying.

Reply:
If we don't auth...then what happens? Does the build fail or does ingestion fail?

Reply:
(and does this still work if building anonymously?)

Reply:
I originally tried without the auth in both the .npmrc and without the task to authenticate and it couldn't get to the packages in the feed. In what instances are we "building anonymously"? I will try to test it out

Reply:
If I was a non-msft dev, does building locally work for me, assuming that the right ingestion has been done already?

Reply:
Let me do some testing

Reply:
(at the same time, this is arcade services, and specifically barviz. Do we care about that scenario?)

Reply:
I am getting "Resource not available for anonymous access" responses. I'll look at the settings of the feed

Reply:
Looks like the npmAuthenticate task isn't actually necessary. https://dev.azure.com/dnceng/internal/_build/results?buildId=1827407&view=logs&j=a5c6a902-5556-5a70-4b67-7f49bfe8ebe3&t=8955ed9e-1866-5780-bb6d-4a3931697224

Reply:
removed.